views/hoc/withRoot.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { MuiThemeProvider } from '@material-ui/core/styles'
+import CssBaseline from '@material-ui/core/CssBaseline'
+
+function withRoot(Component) {
+  function WithRoot(props) {
+    return (
+      <MuiThemeProvider>
+        <CssBaseline />
```

Comment:
can we move these to the top level `App` component?

Reply:
[9d41639](https://github.com/AMP-SCZ/dpdash/pull/317/commits/9d4163949a23685d8c46653ba4f328d0e1dfb8a4)